administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -125,7 +125,7 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL_DESC="An update to Joomla %1$s was
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE="A new version of the Joomla Update Component is available."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_LIVE_UPDATE_DESC="You must update this component first before you can update Joomla! <a class=\"alert-link\" href=\"index.php?option=com_installer&view=update\">Click here to update the component</a>."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="The system is currently checking these plugins to see if they could cause problems during the update.<br><br>Please be patient while the checks are completed."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="I accept the warnings about potentially incompatible extensions and wish to proceed with the update."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Ignore the warnings about potentially incompatible extensions and proceed with the update."
```

You are not necessarily ignoring them.

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Acknowledge the warnings about potentially incompatible extensions and proceed with the update."
```

---

Probably we should then also update the confirmation dialog: https://github.com/joomla/joomla-cms/blob/646d9c7b0bfc11b876c719f6c554c6bfcb2a0be8/administrator/language/en-GB/com_joomlaupdate.ini#L128 ?

---

yes please

I did my first real world update today and this was something I noted to do - you beat me too it.